rpcs3/Crypto/unpkg.cpp

```diff
@@ -45,7 +45,12 @@ bool CheckHeader(const fs::file& pkg_f, PKGHeader* m_header)
 	}
 	}
 
-	if (m_header->header_size != PKG_HEADER_SIZE)
+	if (m_header->header_size == PKG_HEADER_PS1_SIZE)
```

Comment:
I think it's better to allow installing various types of PKGs if the PS3 supports it.